### ISSUE TITLE
Add logic to strip absolute URLs to relative where possible

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#785]
**Is your feature request related to a problem? Please describe.**
This should be done before #784. We should add logic to check if the absolute URL is capable of being a relative URL and change if possible.

**Describe the solution you'd like**
If the URL matches the application URL we should make it relative.

**Testing notes**
Unit testing and cypress tests should be sufficient. Cypress tests validate the HREF but only from '/news/articles/xxxx' onwards. The change in the logic should not require a refactor of the Cypress tests.

**Comment by bcmn** [comment]
A few obvious questions:

Do we still link any relative URLs we bump into? Or don't link them, & force them to be updated.

Presumably we replace both `.com` & `.co.uk`?

The current code doesn't account for different services at all, just `/news` -- I assume this work would be a separate ticket?

**Comment by bcmn** [comment]
Oops, apparently my comment didn't go through -- I spoke with Bogdan:
- Replace both `.com` & `.co.uk`
- Do not do any of the work to support other services.
